lib/ruby-box/item.rb

```diff
@@ -20,15 +20,23 @@ def self.has_many_paginated(*keys)
       keys.each {|key| @@has_many_paginated << key.to_s}
     end
 
-    def move_to( folder_id, name=nil )
+    def move_to(folder_id)
+      url = "#{RubyBox::API_URL}/#{resource_name}/#{id}"
```

Comment:
How about eliminating this variable altogether to avoid confusing `uri` and `url` in the future?

```ruby
url = URI.parse("#{RubyBox::API_URL}/#{resource_name}/#{id}")
```